### DISCUSSION TITLE
view live data updates in the supabase dashboard

### DISCUSSION THREAD

**Discussion by andykais** [supabase/supabase#351] [Feature Requests]
Until I am able to create an admin dashboard myself, the supabase "Table Editor" page has been serving me quite well. However, one feature that would be handy here would be a simple "Refresh" button to get the latest database changes. This is supported indirectly right now by either closing and reopening a table tab in the editor, or by refreshing the web page. Both of these methods feel a bit clunky, and the best alternative I can think of (short of realtime updates) is to give users a refresh button on the table editor page

**Comment by kiwicopple** [comment]
Agreed - the table view should really have live data. We're planning this out now.

While we could use the realtime API that we provide you as a developer, we also plan to give you the ability to "turn off" realtime for certain tables. So we are working on a way to do this, and still recieve realtime updates on the Dashboard.

A refresh button is a good short-term solution. You can also switch tabs (rather than closing/opening).
